apps/meteor/client/components/Omnichannel/Tags.tsx

```diff
@@ -70,6 +70,7 @@ const Tags = ({
 			<Field.Label required={tagRequired} mb='x4'>
 				{t('Tags')}
 			</Field.Label>
+
```

Comment:
Accidental changes?

Reply:
Intentional actually. Improves the readability in my opinion.